.gitignore

```diff
@@ -43,3 +43,5 @@ hs_err_pid*
 
 # Ignore mac attributes files
 *.DS_STORE
+# Ignore bazel-deps resolverCache. See: https://github.com/johnynek/bazel-deps#options
```

Comment:
do you have experience with `bazel_output_base` from that link?

Reply:
We use `bazel_output_base` internally. @kevingessner added it [here](https://github.com/johnynek/bazel-deps/commit/a88479c0123a021b352888b7e9bd3cd46e69a7a8). I like it and I think it fits in with Bazel conventions nicely (e.g. removed with `bazel clean --expunge`). I can switch it in the PR if you like.

Reply:
mmm... actually let's leave it like that, in case it turns out that it's useful to only have Bazel stuff in `bazel-out`.